security/pfSense-pkg-suricata/files/usr/local/pkg/suricata/suricata_generate_yaml.php

```diff
@@ -338,8 +338,19 @@
 
 // EVE log output included information
 $eve_out_types = "";
-if ($suricatacfg['eve_log_alerts'] == 'on')
+if (($suricatacfg['eve_log_alerts'] == 'on') && ($suricatacfg['eve_log_alerts_payload'] == 'off'))	
 	$eve_out_types .= "\n        - alert";
+	
+if (($suricatacfg['eve_log_alerts'] == 'on') && ($suricatacfg['eve_log_alerts_payload'] == 'on')) {
+	$eve_out_types .= "\n        - alert:";
+	$eve_out_types .= "\n            payload: yes # enable dumping payload in Base64";
+	$eve_out_types .= "\n            payload-printable: yes # enable dumping payload in printable (lossy) format";
+	$eve_out_types .= "\n            packet: yes # enable dumping of packet (without stream segments)";
+	$eve_out_types .= "\n            http: yes # enable dumping of http fields";
+	$eve_out_types .= "\n            tls: yes # enable dumping of tls fields";
+        $eve_out_types .= "\n            ssh: yes # enable dumping of ssh fields";
+        $eve_out_types .= "\n            smtp: yes # enable dumping of smtp fields";
+		}
```

Comment:
Almost there. ssh and smtp lines are still indented with spaces and } should be at line begin